docs/kop.md

```diff
@@ -67,16 +67,20 @@ After you copy the `.nar` file to your Pulsar `/protocols` directory, you need t
     messagingProtocols=kafka
     protocolHandlerDirectory=./protocols
     allowAutoTopicCreationType=partitioned
+    narExtractionDirectory=/path/to/nar
     ```
 
     | Property | Default value | Proposed value |
     | :------- | :---------------------------- | :------------ |
     | `messagingProtocols` |  | kafka |
     | `protocolHandlerDirectory`|./protocols  | Location of KoP NAR file |
     | `allowAutoTopicCreationType`| non-partitioned | partitioned |
+    | `narExtractionDirectory` | `/tmp/pulsar-nar` | Location of unpacked KoP NAR file |
 
     By default, `allowAutoTopicCreationType` is set to `non-partitioned`. Since topics are partitioned by default in Kafka, it's better to avoid creating non-partitioned topics for Kafka clients unless Kafka clients need to interact with existing non-partitioned topics.
 
+    It's also recommended to configure `narExtractionDirectory` to another path because the KoP NAR file will be unpacked under this directory. However, the default `/tmp/pulsar-nar` directory is under the `/tmp` directory so some classes could be automatically deleted by the system. In this case, `ClassNotFoundException` or `NoClassDefFoundError` could be raised.
```

Comment:
```suggestion
    It is recommended to configure the `narExtractionDirectory` option to another path because the KoP NAR file will be unpacked under this directory. However, the default `/tmp/pulsar-nar` directory is under the `/tmp` directory so some classes could be automatically deleted by the system. In this case, a`ClassNotFoundException` or `NoClassDefFoundError` error could be raised.
```
What should we do to avoid this error? To set the `narExtractionDirectory` to another path?

Reply:
Yes.